assets/src/components/kubernetes/audit/Audit.tsx

```diff
@@ -0,0 +1,95 @@
+import { Table } from '@pluralsh/design-system'
+import { CellContext, createColumnHelper } from '@tanstack/react-table'
+import { useEffect, useMemo } from 'react'
+import {
+  ClusterAuditLog,
+  useKubernetesClusterAuditLogsQuery,
+} from '../../../generated/graphql.ts'
+import { GqlError } from '../../utils/Alert.tsx'
+import { ScrollablePage } from '../../utils/layout/ScrollablePage.tsx'
+import { DateTimeCol } from '../../utils/table/DateTimeCol.tsx'
+import { useFetchPaginatedData } from '../../utils/table/useFetchPaginatedData.tsx'
+import UserInfo from '../../utils/UserInfo.tsx'
+import { useCluster } from '../Cluster.tsx'
+import { useDataSelect } from '../common/DataSelect.tsx'
+
+const columnHelper = createColumnHelper<ClusterAuditLog>()
+
+const RawColumn = ({ getValue }: CellContext<any, string>): string => getValue()
+
+const pathColumn = columnHelper.accessor((log) => log?.path, {
+  id: 'path',
+  header: 'Path',
+  enableSorting: true,
+  cell: RawColumn,
+})
+
+const methodColumn = columnHelper.accessor((log) => log?.method, {
+  id: 'method',
+  header: 'Method',
+  enableSorting: true,
+  cell: RawColumn,
+})
+
+const insertedAtColumn = columnHelper.accessor((log) => log?.insertedAt, {
+  id: 'timestamp',
+  header: 'Timestamp',
+  enableSorting: true,
+  cell: ({ getValue }) => <DateTimeCol date={getValue()} />,
+})
+
+const userColumn = columnHelper.accessor((log) => log?.actor, {
+  id: 'user',
+  header: 'User',
+  enableSorting: true,
+  cell: ({ getValue }) => <UserInfo user={getValue() ?? {}} />,
+})
+
+const columns = [userColumn, pathColumn, methodColumn, insertedAtColumn]
```

Comment:
super NIT-y, but I think the order here should be [method, path, actor, timestamp]